README.md

```diff
@@ -15,8 +15,8 @@ suitable for more general usage.
 
 # Status
 
-MLIR has Cuda and Vulkan support disabled. LLVM does not yet have configuration
-detection and hardcodes values in the config.
+MLIR has Cuda support and the mlir-vulkan-runner disabled. LLVM does not yet
+have configuration detection and hardcodes values in the config.
```

Comment:
It isn't clear to me if this is already the case, but can the `vulkan_sdk_setup` routine be used by consumers of the repo? If not, could you rearrange things to make that work?

Then, maybe it makes sense to document how users interested in enabling Vulkan rules can use your repository rule from their `WORKSPACE` file (unless they want to write their own)?

Reply:
Minor point, but I'd give some examples here... just feel like it'll be easier for users than finding the demos. But not a big deal, also would be fine to follow-up with docs later.

Reply:
You mean a usage demo? I can do that, but a demo that builds is always preferable to a demo in documentation that gets outdated IMO. Seems orthogonal to this PR regardless so going to defer that.